webpack.config.js

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
+const TerserPlugin = require('terser-webpack-plugin');
+
+const nodeEnv = process.env.NODE_ENV || 'development';
```

Comment:
Using the environment variable is obsolete. One could determine the mode by grabbing the argumment like `const mode = process.argv.includes('--mode=production') ? 'production' : 'development';`. And than `isProd` simple can be replaced by `mode === 'production'` - no real benefit from adding an extra variable here.